README.md

```diff
@@ -46,6 +48,23 @@ Les sources de données (PDF ou site web) sont notées dans chaque fichier YAML.
 
 Consigne générale : le nombre d'espaces en début de ligne est très important, ainsi que la position des tirets `-`, soyez vigilant en complétant les fichiers.
 
+#### Creer un fichier YAML
+
+Pour creer un fichier YAML vous devrez faire votre propre copie du 
+repository, qui sera modifiable pour vous, et depuis laquel vous
+pourrez soumetre vos `pull requests`:
+
+  - (A la premiere utilisation) Faites une copie (`fork`; en haut a droite)
+  - Dans le nouveau repository ajoutez vos fichiers dans les sous dossier 
+  correspondant aux regions/departement de vos données.
+  - Creer une pull request pour ajouter vos données au parent;
```

Comment:
Faites une PR par fichier Yaml, nommez votre PR selon le format 'source - date'. Par exemple : ARS Corse - 15/03

Reply:
Ok dans ce cas il faut modifier les conseils d'utilisation, il faut que chaque personne face miroir du repo d'origine en faisant des branches.

Par exemple dans mon cas j'aurais du créer:
 + `payoto:ars-guadeloupe-2020-03-20` (depuis master)
 + Que je dois ensuite PR sur le repo d'origine

Ça devient compliqué, est ce que j'ajuste l'explication?

Reply:
> Ok dans ce cas il faut modifier les conseils d'utilisation, il faut que chaque personne face miroir du repo d'origine en faisant des branches.
> 
> Par exemple dans mon cas j'aurais du créer:
> 
>     * `payoto:ars-guadeloupe-2020-03-20` (depuis master)
> 
>     * Que je dois ensuite PR sur le repo d'origine
> 
> 
> Ça devient compliqué, est ce que j'ajuste l'explication?

Je crois c'est plus simple de créer le yaml dans git et faire un PR. je fork>comit>pr>merge ça parait long et peut sombre à un nouveau.

Reply:
C'est pas si compliqué, github prend tout en charge, tu fais new file et il te laisse créer une branche direct.

Reply:
Ok top, merci pour le travail. je suis dispo le WE pour un coup de main s'il y a des choses à ajouter.